code/modules/martial_arts/cqc.dm

```diff
@@ -168,21 +168,31 @@
 	if(!can_use(A))
 		return FALSE
 	add_to_streak("D", D)
+	var/obj/item/I = null
 	if(check_streak(A, D))
 		return TRUE
-	if(!restraining)
-		D.visible_message("<span class='warning'>[A] strikes [D]'s jaw with their hand!</span>", \
-							"<span class='userdanger'>[A] strikes your jaw, disorienting you!</span>")
-		playsound(get_turf(D), 'sound/weapons/cqchit1.ogg', 50, 1, -1)
-		D.Jitter(2)
-		D.apply_damage(5, BRUTE)
-		add_attack_logs(A, D, "Melee attacked with martial-art [src] : Disarm", ATKLOG_ALL)
-	if(restraining)
+	if(prob(65))
```

Comment:
Unless you also fixed the bug with this disarm implementation, which I don't think you did, this'll cause CQC disarms to disarm a target twice for each click : One normal disarm, followed by one CQC disarm. I removed part of the disarm so that CQC would damage on disarm then proceed with a regular disarm proc. Ideally i'd prefer if the bug that causes the double disarm was fixed, but in the meantime this proc needs to stay as a damage buff only.

Reply:
The bug has been fixed.